src/renderers/common/XRManager.js

```diff
@@ -651,7 +653,9 @@ class XRManager extends EventDispatcher {
 						format: RGBAFormat,
 						type: UnsignedByteType,
 						colorSpace: renderer.outputColorSpace,
-						stencilBuffer: renderer.stencil
+						stencilBuffer: renderer.stencil,
+						resolveDepthBuffer: ( glBaseLayer.ignoreDepthValues === false ),
```

Comment:
The WebGL 2 backend of `WebGPURenderer` does not yet honor the `resolve*` flags. That needs to be fixed with a different PR.

Reply:
yes. The XR part is also using `ignoreDepthValues` to not pass the depth buffer and use that as a signal to ignore it in the renderer. That should also be updated.